tetris.py

```diff
@@ -27,26 +28,50 @@ def run_gui():
     root = tkinter.Tk()
     root.resizable(False, False)
 
-    topbar = tkinter.Label(root)
-    topbar["bg"] = D_GREY
-    topbar.grid(row=0, column=0, sticky="we")
-    game_canvas = tkinter.Canvas(
+    game_board = tkinter.Canvas(
         root,
         width=square_size * game_width,
         height=square_size * game_height,
-        highlightthickness=0,
     )
-    game_canvas.grid()
+    game_board.grid(row=1, sticky='nswe')
 
-    tetris_gui = TetrisGUI(game_speed, game_canvas)
+    topbar = tkinter.Frame(root, bg=D_GREY, relief = 'ridge')
+    topbar.grid(row=0, columnspan=2, sticky='we')
 
-    root.bind("<Left>", tetris_gui.left_mediator)
-    root.bind("<Right>", tetris_gui.right_mediator)
-    root.bind("<Up>", tetris_gui.rotate_mediator)
+    topbar_time = tkinter.Label(topbar, bg=D_GREY, text='00:00', font = 'digital-7', fg='orange', borderwidth=1)
+    topbar_time.pack(side='left', padx=10)
 
-    tetris_gui.draw_board()
-    tetris_gui.draw_block()
-    tetris_gui.block_mediator()
+    topbar_score = tkinter.Label(topbar, bg=D_GREY, text='foo', font = 'digital-7', fg='orange', borderwidth=1)
+    topbar_score.pack(side='left', fill='x', expand=True)
+
+    topbar_board = tkinter.Canvas(topbar, bg=D_GREY, width = square_size*4, height = square_size*2, highlightthickness=0)
+    topbar_board.pack(side='right', expand=True)
+
+    sidebar = tkinter.Frame(root, bg=D_GREY)
+    sidebar.grid(row=1, column=1, sticky='nsw')
+
+    new_game_button = tkinter.Button(sidebar, text = 'start')
+    new_game_button.grid(sticky='n')
+
+    new_game_button2 = tkinter.Button(sidebar, text = 'start')
+    new_game_button2.grid(sticky='n')
+
+    new_game_button3 = tkinter.Button(sidebar, text = 'start')
+    new_game_button3.grid(sticky='n')
+
+    tetris_gui = TetrisGUI(game_speed, game_board)
+
+    main_board = Board(game_board, GREY, game_width, game_height, random.choice(shape_names), (int(game_width / 2), -2), 0)
```

Comment:
Variable names are confusing here: `game_board` is not actually a board, but a canvas.

Reply:
I've changed to be the same as topbar_board. So I should do it the opposite way and name topbar_board topbar_canvas instead?